### PR TITLE
Include temperature in Output_L1Error()

### PR DESCRIPTION
**Pull request by hyschive**

### CODE REVIEW

File: src/Output/Output_L1Error.cpp [hunk 1]
```diff
@@ -319,20 +331,25 @@ void WriteFile( void (*AnalFunc_Flu)( real fluid[], const double x, const double
...
 
-   Nume[ENGY] = Hydro_Con2Pres( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
-                                CheckMinPres_No, NULL_REAL, Emag_Nume,
-                                EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real  Pres_Nume       = Hydro_Con2Pres( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
+                                                 CheckMinPres_No, NULL_REAL, Emag_Nume,
+                                                 EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real  Temp_Nume       = Hydro_Con2Temp( Nume[DENS], Nume[MOMX], Nume[MOMY], Nume[MOMZ], Nume[ENGY], Nume+NCOMP_FLUID,
+                                                 CheckMinTemp_No, NULL_REAL, Emag_Nume,
+                                                 EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table );
```
Comment (BarryTC) [inline src/Output/Output_L1Error.cpp:350]: **_EoS_DensEint2Pres_CPUPtr_** should be changed to **_EoS_DensEint2Temp_CPUPtr_**

File: src/Output/Output_L1Error.cpp [hunk 2]
```diff
@@ -347,14 +364,18 @@ void WriteFile( void (*AnalFunc_Flu)( real fluid[], const double x, const double
...
 
-   Anal[ENGY] = Hydro_Con2Pres( Anal[DENS], Anal[MOMX], Anal[MOMY], Anal[MOMZ], Anal[ENGY], Anal+NCOMP_FLUID,
-                                CheckMinPres_No, NULL_REAL, Emag_Zero,
-                                EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real Pres_Anal = Hydro_Con2Pres( Anal[DENS], Anal[MOMX], Anal[MOMY], Anal[MOMZ], Anal[ENGY], Anal+NCOMP_FLUID,
+                                          CheckMinPres_No, NULL_REAL, Emag_Zero,
+                                          EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table, NULL );
+   const real Temp_Anal = Hydro_Con2Temp( Anal[DENS], Anal[MOMX], Anal[MOMY], Anal[MOMZ], Anal[ENGY], Anal+NCOMP_FLUID,
+                                          CheckMinTemp_No, NULL_REAL, Emag_Zero,
+                                          EoS_DensEint2Pres_CPUPtr, EoS_AuxArray_Flt, EoS_AuxArray_Int, h_EoS_Table );
```
Comment (BarryTC) [inline src/Output/Output_L1Error.cpp:376]: **_EoS_DensEint2Pres_CPUPtr_** should be changed to **_EoS_DensEint2Temp_CPUPtr_**

**Review by BarryTC — CHANGES_REQUESTED** [review]
Two minor corrections to the otherwise ready-to-go updated script.

File: src/Output/Output_L1Error.cpp (hunk 1 above)
Comment (hyschive) [reply src/Output/Output_L1Error.cpp]: Good catch! Fixed.

File: src/Output/Output_L1Error.cpp (hunk 2 above)
Comment (hyschive) [reply src/Output/Output_L1Error.cpp]: Fixed

**Review by BarryTC — APPROVED** [review]

**Comment by hyschive** [comment]
@BarryTC Thanks for helping review it!

**Pull request merged**